rfc/helm.md

```diff
@@ -0,0 +1,131 @@
+Helm Chart Support
+====================
+
+The current CLI install process uses Helm libraries under the hood, but just as a template library. There is currently a Helm chart under the `charts` directory that allows installing the control plane, but uninjected, and there's no `values.yaml` file provided.
+                                                                                                                                                                                                                                             
+The intention here is to provide a new chart with an injected control plane so that users can install linkerd through a simple `helm install incubator/linkerd2` command.
+                                                                                                                                                                                                                                             
+Helm charts can't rely on go code besides the functions provided by the go template library, the Sprig library, and a few extra functions provided by Helm itself. This implies a few compromises:
+                                                         
+- We can't validate the install options provided in `values.yaml`. Instead, a new set of `linkerd check` checks could help catching invalid options, post-install.
+- We should provide a comprehensive `values.yaml` file that would contain the most common settings, but heavily annotated to instruct users about alternate settings for advanced scenarios.                                                              
```

Comment:
On top of a heavily annotated, comprehensive set of defaults, I would also like us to put some “configurations” together such as what `—ha` does today.